docs/dictionary.rst

```diff
@@ -0,0 +1,1101 @@
+
+Further reading: `<https://www.nih.gov/about-nih/what-we-do/nih-almanac/national-cancer-institute-nci>`_  
+
+.. updated 2022-05-16  
+
+
+
+.. _dict NCPI:
+
+NCPI
+----
+*abbreviation for* NIH Cloud Platform Interoperability  
+
+	An effort to connect five :ref:`dict NIH` cloud projects and ensure they are interoperable. The five projects covered under this are the :ref:`dict AnVIL Project`, :ref:`dict BioData Catalyst`, Cancer Research Data Commons, Kids First, and the National Center for Biotechnology Information.  
+
+.. note:: This term as we define it here is associated with NIH and may have different definitions in other contexts.  
+
+Further reading: `<https://datascience.nih.gov/nih-cloud-platform-interoperability-effort>`_  
+
+.. updated 2022-05-16  
+
+
+
+.. _dict Nextflow:
+
+Nextflow
+--------
+	A Java-based computational workflow engine. Dockstore supports the hosting of Nextflow workflows.  
+
+Further reading: `<https://www.nextflow.io/>`_  
+
+.. updated 2022-05-16  
+
+
+
+.. _dict NFL:
+
+NFL
+---
+*abbreviation for* :ref:`dict Nextflow`  
+
+	An uncommon acronym for :ref:`dict Nextflow`. This abbreviation is not used as frequently as :ref:`dict CWL` or :ref:`dict WDL`, but does see usage occasionally.  
+
+
+.. updated 2022-05-16  
+
+
+
+.. _dict NHGRI:
+
+NHGRI
+-----
+*abbreviation for* National Human Genome Research Institute  
+
+	A division of the :ref:`dict NIH` that focus on genomics research. Funds the :ref:`dict AnVIL Project`.  
+
+Further reading: `<https://www.genome.gov/>`_  
+
+.. updated 2022-05-16  
+
+
+
+.. _dict NHLBI:
+
+NHLBI
+-----
+*abbreviation for* National Heart, Lungs, and Blood Institute  
+
+	A division of the :ref:`dict NIH` that focuses on heart, lung, blood, and sleep health. Funds the :ref:`dict BioData Catalyst` platform.  
+
+Further reading: `<https://www.nhlbi.nih.gov/>`_  
+
+.. updated 2022-05-16  
+
+
+
+.. _dict NIH:
+
+NIH
+---
+*abbreviation for* National Institute of Health  
+
+	An American government institution, part of the Department of Health and Human Services (HHS), that engages in medical research.  
+
+Further reading: `<https://www.nih.gov/>`_  
+
+.. updated 2022-05-16  
+
+
+
+.. _dict OICR:
+
+OICR
+----
+*abbreviation for* Ontario Institute for Cancer Research  
+
+	A non-profit research institute based in Toronto that is focused on cancer detection and treatment. One of the two institutes involved in the development of Dockstore, the other being :ref:`dict UCSC`.  
+
+Further reading: `<https://oicr.on.ca/>`_  
+
+.. updated 2022-05-16  
+
+
+
+.. _dict ORCID:
+
+ORCID
+-----
+pronounced "or-kid", rhymes with kid  
+
+*abbreviation for* Open Researcher and Contributor ID  
+
+	ID used to identify researchers and their work in a way that doesn't solely rely on names.  
+
+Further reading: `<https://info.orcid.org/what-is-orcid/>`_  
+
+.. updated 2022-05-16  
+
+
+
+.. _dict organization:
+
+organization
+------------
```

Comment:
Users also should have the authority to speak for the institute, grant, etc. in a technical manner

Reply:
Does this apply to the first sentence or the second? ie are users within an organization have that authority, or only users who are applying to have an organization created?

Reply:
I think the latter. We don't really check on the former and there are different roles anyway, some of which clearly have lesser authority.